.github/workflows/deploy.yml

```diff
@@ -3,23 +3,15 @@ name: Deploy
 on:
   push:
     branches: [ 'develop', 'master' ]
-  pull_request:
-    branches: [ 'develop', 'master' ]
 
-jobs:
-  cancel-previous:
-    name: 'Cancel Previous Runs'
-    runs-on: ubuntu-latest
-    timeout-minutes: 3
-    steps:
-      - uses: styfle/cancel-workflow-action@0.8.0
-        with:
-          access_token: ${{ github.token }}
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
```

Comment:
Could you please elaborate your reasoning begind cancelling running develop deploy?

Reply:
Thanks for pointing, no deploy should be cancelled either for develop or master.